client/app/core/navigation.service.js

```diff
@@ -1,12 +1,22 @@
+import NavActions from '../actions/nav'
+/* eslint no-unused-vars: 0 */
+
 /** @ngInject */
-export function NavigationFactory (RBAC, Polling, POLLING_INTERVAL, CollectionsApi) {
+export function NavigationFactory (RBAC, Polling, POLLING_INTERVAL, $ngRedux, CollectionsApi) {
   let menuItems = []
   var service = {
     get: getNavigation,
     menuItems: menuItems,
     init: initNavigation,
     updateBadgeCounts: updateBadgeCounts
   }
+  const actions = NavActions
+  const mapStateToThis = function (state) {
+    return { navCount: state.nav }
+  }
```

Comment:
can be written as:
```js
const mapStateToThis = ({ nav }) => ({
  navCount: nav,
});
```

Reply:
@ohadlevy , these syntax changes look great.  Do you know what version of ES they were introduced.  I wanted to sit down and read up on all the different syntactical options just so I can start taking more advantage of them in the future

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment